plugins/org.jkiss.dbeaver.data.transfer.ui/src/org/jkiss/dbeaver/tools/transfer/ui/pages/stream/StreamConsumerPageSettings.java

```diff
@@ -415,6 +421,35 @@ protected void setValue(Object element, Object value) {
                 column.getColumn().setText(DTUIMessages.stream_consumer_page_mapping_mapping_column_name);
             }
 
+            Composite panel = new Composite(group, SWT.NONE);
+            panel.setLayout(new FillLayout());
+            Button selectAllButton = new Button(panel, SWT.NONE);
```

Comment:
Please consider using `UIUtils.createDialogButton` instead. They have padding and generally look better.